pkg/lib/serverTime.js

```diff
@@ -106,7 +106,7 @@ export function ServerTime() {
     self.wait = function wait() {
         if (remote_offset === null)
             return self.update();
-        return cockpit.resolve();
+        return Promise.resolve();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18487-20230310-143650-f841503b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)